.github/actions/unit-tests/action.yml

```diff
@@ -19,14 +19,14 @@ runs:
       run: | 
         source ~/.bashrc
         conda clean -i -y
-        ./scripts/configure_conda researcher
+        ./scripts/configure_conda researcher node
```

Comment:
`node` should not be installed in actions for unit tests, as unit tests don't use this environment (by convention it uses `researcher`)

If needed to MNIST test, its installation should be moved there.

Pushing a patch for this.